doc/source/user/how-to-verify-bug.rst

```diff
@@ -149,15 +151,17 @@ To verify the fix:
     >>> np.polymul(a.astype(np.complex64), z.astype(np.complex64)).dtype
     dtype('complex64')
 
-Note that the correct ``dtype`` is now returned even when the zero array is one
-of the arguments to `numpy.polymul`.
+Note that the correct ``dtype`` is now returned even when a zero array is one of
+the arguments to `numpy.polymul`.
 
-***********************************************************************
-5. Support the NumPy development community by verifying and fixing bugs
-***********************************************************************
+*********************************************************
+1. Support NumPy development by verifying and fixing bugs
```

Comment:
```suggestion
5. Support NumPy development by verifying and fixing bugs
```

Although actually I'd really rather not have the numbering in the titles, I don't think they're there anywhere else in the docs.

Reply:
In this case I'd actually advocate for the numbering, I think it makes the instructions easier to follow.